components/PassedQuestions/PassedQuestions.tsx

```diff
@@ -4,10 +4,13 @@ import {
 } from '@/hooks/useContractBuilder'
 import { useContractBuilderProvider } from '@/providers/ContractBuilderProvider'
 
-const PassedQuestions = () => {
+interface PassedQuestionsProps {
+  textSize?: string; // Optional prop
```

Comment:
Please remove dev comments. 
```suggestion
  textSize?: string;
```